lib/modules/platform/bitbucket/index.spec.ts

```diff
@@ -95,6 +95,15 @@ describe('modules/platform/bitbucket/index', () => {
       ).toMatchSnapshot();
     });
 
+    it('should init with only token', async () => {
+      httpMock.scope(baseUrl).get('/2.0/user').reply(200);
+      expect(
+        await bitbucket.initPlatform({
+          token: 'token',
+        })
+      ).toMatchSnapshot();
```

Comment:
Use `toEqual` matcher instead. Don't follow other tests and read our best practices guid 🙃 

https://github.com/renovatebot/renovate/blob/main/docs/development/best-practices.md#unit-testing